src/utils/Calendar.js

```diff
@@ -0,0 +1,23 @@
+export const getNewDateStateChange = ({ code, focusedDay, startDate, endDate }) => {
```

Comment:
now that this is more specific to state changes within the component, I think it would be valuable to keep it in the same file and export it from there for testing.

Reply:
okay so check out what i pushed up - now instead of exporting the calendar like this: 
`module.exports = Radium(Calendar);` I've exported it like this: `export default Radium(Calendar)` and then I also export the func at the top for testing. Sam helped me make this work by importing using ES6 instead of commonjs